src/cli/parse-ssl-options.js

```diff
@@ -0,0 +1,53 @@
+import fs from 'fs';
+import os from 'os';
+
+const MAX_PATH_LENGTH = os.type() === 'Windows_NT' ? 260 : Infinity;
+
+const OPTIONS_SEPARATOR          = ';';
+const OPTION_KEY_VALUE_SEPARATOR = '=';
+const FILE_OPTION_NAMES          = ['cert', 'key', 'pfx'];
+const NUMBER_REG_EX              = /^[0-9-.,]+$/;
+const BOOLEAN_STRING_VALUES      = ['true', 'false'];
+
+export default function (optionsStr = '') {
+    const splittedOptions = optionsStr.split(OPTIONS_SEPARATOR);
+
+    if (!splittedOptions.length)
+        return null;
+
+    const parsedOptions = {};
+
+    splittedOptions.forEach(item => {
+        const keyValuePair = item.split(OPTION_KEY_VALUE_SEPARATOR);
+        const key          = keyValuePair[0];
+        let value          = keyValuePair[1];
+
+        if (!key || !value)
+            return;
+
+        value = convertToBestFitType(value);
+
+        if (FILE_OPTION_NAMES.includes(key) && value.length < MAX_PATH_LENGTH)
+            value = fs.readFileSync(value);
```

Comment:
On Linux `MAX_PATH_LENGTH` is 4096 (4kB). My SSH private key is about 1.5kB. If I pass my certificate, I will get an error from `fs.readFileSync`. We should check if the file exists, and if it don't, also assume that value is a buffer.